File: src/main/java/info/dddeurope/lab/app/domain/PostAggregate/Post.java
package info.dddeurope.lab.app.domain.PostAggregate;

import info.dddeurope.lab.app.domain.UserAggregate.Age;
import lombok.Data;

@Data
public class Post {

    String id;
    String publisherId;
    String title; 
    String body;
    Age ageLimit;

    public Post(String id, String publisherId, String postTitle, String postBody, Age ageLimit) {
        this.id = id;
        this.publisherId = publisherId;
        this.title = postTitle;
        this.body = postBody;
        this.ageLimit = ageLimit;
    }

}

File: DotNet/App/Domain/PostAggregate/Post.cs
using App.Domain.UserAggregate;

namespace App.Domain.PostAggregate
{
    public class Post
    {
        public string Id { get; }
        public string PublisherId { get; }
        public string PostTitle { get; }
        public string PostBody { get; }
        public Age AgeLimit { get; }

        public Post(string id, string publisherId, string postTitle, string postBody, Age ageLimit)
        {
            Id = id;
            PublisherId = publisherId;
            PostTitle = postTitle;
            PostBody = postBody;
            AgeLimit = ageLimit;
        }
    }
}